src/codeu/chat/server/ServerInfo.java

```diff
@@ -0,0 +1,49 @@
+package codeu.chat.server;
+
+import codeu.chat.util.Time;
+import codeu.chat.util.Uuid;
+
+import java.io.IOException;
+
+/**
+ * Created by himonal on 6/2/17.
+ */
+public class ServerInfo {
+
+    public final Time startTime;
+    private final static String SERVER_VERSION = "1.0.0";
+    public final Uuid version;
+
+    public ServerInfo() {
+        this.startTime = Time.now();
+        Uuid tempVersion = null;
+
+        try {
+            tempVersion = Uuid.parse(SERVER_VERSION);
+        }
+        catch (IOException e) {
+            System.out.println("Wrong");
+        }
+
+        this.version = tempVersion;
+    }
+
+    public ServerInfo(Time startTime) {
```

Comment:
It looks like you've combined steps from the Version Check tutorial and the Uptime tutorial - I'd remove these changes and try only following the steps after "Up Time" on page 6 and before "Tokenizing Input" on page 7